clientv3/config.go

```diff
@@ -41,6 +42,13 @@ type Config struct {
 	// Password is a password for authentication.
 	Password string `json:"password"`
 
+	// UseLatestCluster when set will refuse to create a client against an outdated cluster.
+	UseLatestCluster bool `json:"use-latest-cluster"`
```

Comment:
EnsureClusterVersion? Or CheckClusterVersion? `use` is not a strong enough word I feel.